api/v1alpha1/mcp_route.go

```diff
@@ -161,6 +161,14 @@ type MCPBackendAPIKey struct {
 	//
 	// +optional
 	Inline *string `json:"inline,omitempty"`
+
+	// Header is the HTTP header to inject the API key into.
+	// If not specified, the default is "Authorization".
```

Comment:
let's document that when this is "authorization" the value is prefixed with "bearer"